pkg/disttask/importinto/planner.go

```diff
@@ -378,6 +390,20 @@ func generateWriteIngestSpecs(planCtx planner.PlanCtx, p *LogicalPlan) ([]planne
 		}
 		specs = append(specs, specsForOneSubtask...)
 	}
+	// write external meta to storage when using global sort
+	if planCtx.GlobalSort {
+		for i, spec := range specs {
+			if w, ok := spec.(*WriteIngestSpec); ok {
+				externalPath := writeIngestStepExternalMetaPath(planCtx.TaskID, i+1)
+				if err := external.WriteJSONToExternalStorage(planCtx.Ctx, controller.GlobalSortStore, externalPath, w.WriteIngestStepExternalMeta); err != nil {
```

Comment:
if the file already exist, it's overwrite silently?

Reply:
yes, overwrite when rerun.